### PR TITLE
Add null test for Nullable GetUnderlyingType

### PR DESCRIPTION
**Pull request by ianhays**

### CODE REVIEW

File: src/System.Runtime/tests/System/Nullable.cs [hunk 1]
```diff
@@ -77,6 +77,12 @@ public static void TestGetUnderlyingType()
     }
 
     [Fact]
+    public static void TestGetUnderlyingType_NullTypeThrowsArgumentNullException()
+    {
+        Assert.Throws<ArgumentNullException>(() => Nullable.GetUnderlyingType((Type)null));
```
Comment (JonHanna) [inline src/System.Runtime/tests/System/Nullable.cs]: `Assert.Throws<ArgumentNullException>("nullableType", () => Nullable.GetUnderlyingType((Type)null));`?

Comment (ianhays) [inline src/System.Runtime/tests/System/Nullable.cs]: Thanks, I always forget the parameter names.

File: src/System.Runtime/tests/System/Nullable.cs [hunk 2]
```diff
@@ -3,7 +3,7 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
-
+using System.Diagnostics;
```
Comment (JonHanna) [inline src/System.Runtime/tests/System/Nullable.cs]: Nit: it doesn't seem like you needed to add this directive.

**Comment by JonHanna** [comment]
LGTM

**Comment by ianhays** [comment]
Thanks, Jon! Removed the unnecessary using and rebased.

**Comment by mmitche** [comment]
@dotnet-bot test this please

**Comment by JonHanna** [comment]
Test Innerloop Windows_NT Debug Build and Test
(Two networking-related tests at http://dotnet-ci.cloudapp.net/job/dotnet_corefx/job/windows_nt_debug_prtest/4361/ that may be just a network glitch).

**Pull request merged**